logstash-core-plugin-api/logstash-core-plugin-api.gemspec

```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
```

Comment:
To the full disclosure I copied one of our gemspec to make it work and in theses we actually use this syntax.

On a side node there is effectively 3 way that I know to specify the encoding in ruby files:

```
# This is use by newer version of `bundler gem`
# coding: utf-8
# encoding: utf-8
# -*- encoding: utf-8 -*-
```